_layouts/project-index.html

```diff
@@ -168,6 +168,7 @@ <h2 class="boxed">See more projects by impact area</h2>
 
   <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.js'></script>
   <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.css' rel='stylesheet' />
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
```

Comment:
@ramyaragupathy Do we want to consider hosting these type of resources ourselves? related to https://github.com/hotosm/tasking-manager/issues/912

Reply:
Good point - similar to our tracking code. I feel this might optimise page load. Will coordinate with @dakotabenjamin to get this done